x/posts/internal/types/post.go

```diff
@@ -260,6 +260,26 @@ func (p Post) ContentsEquals(other Post) bool {
 		ArePollDataEquals(p.PollData, other.PollData)
 }
 
+// GetPostHashtags returns all the post's hashtags without duplicates
+func (p Post) GetPostHashtags() []string {
+	hashtags := HashtagRegEx.FindAllString(p.Message, -1)
+	return Unique(hashtags)
+}
+
+// Unique returns the given input slice without any duplicated value inside it
+func Unique(input []string) []string {
```

Comment:
Can we move this outside of here? Maybe inside some `common.go` file or something? This might be useful somewhere else as well